xdsl/dialects/riscv.py

```diff
@@ -2000,14 +2000,14 @@ def my_add():
     name = "riscv.label"
     label: LabelAttr = attr_def(LabelAttr)
     comment: StringAttr | None = opt_attr_def(StringAttr)
-    data: OptRegion = opt_region_def("single_block")
+    data: OptRegion = opt_region_def()
 
     traits = frozenset([NoTerminator()])
```

Comment:
Does this trait work with multi-block regions? I don't think so, right?
Shouldn't this be removed? Which will then require terminators in all contained blocks?

Reply:
Ah I didn't realise it wouldn't I removed it in the previous PR but for this one thought it might be ok without it. Maybe I should just reopen the old one, will take a look.